src/main/java/seedu/address/logic/parser/LockCommandParser.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.logic.parser;
+
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.LockCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public class LockCommandParser implements Parser<LockCommand> {
+
+    @Override
+    public LockCommand parse(String args) throws ParseException {
+        String trimmedArgs = args.trim();
+        String[] passwords = trimmedArgs.split("\\s+");
+
+        if (passwords.length < 1 || passwords.length > 2) {
```

Comment:
The code here does not capture the error where user input "lock" has no password.

Reply:
Thanks for the feedback! Turns out String.split() will return an array with 1 element when the string is an empty string. A similar problem in UnlockCommandParser has also been fixed.